### PR TITLE
shortint/fix degree

### PR DESCRIPTION
**Pull request by mayeul-zama**
In shortint, change the meaning of degree to be a measure to guarantee both:
- no padding-bit overflow
- not breaking PBS noise limit

### CODE REVIEW

**Comment by mayeul-zama** [comment]
@slab-ci cpu_fast_test

File: tfhe/src/shortint/server_key/mod.rs [hunk 1]
```diff
@@ -274,7 +274,7 @@ fn ciphertexts_can_be_packed_without_exceeding_space(
 #[must_use]
 pub struct LookupTable<C: Container<Element = u64>> {
     pub acc: GlweCiphertext<C>,
-    pub degree: Degree,
+    pub max_value: u64,
```
Comment (IceTDrinker) [inline tfhe/src/shortint/server_key/mod.rs:277]: disagree strongly with this change

**Review by IceTDrinker — COMMENTED** [review]
As discussed I believe this change would be better in conjunction with the noise update

**Comment by IceTDrinker** [comment]
can this be put in draft or closed for now as we prefer having the noise rework at the same time ? 🙂

**Pull request closed without merging**